clients/src/main/java/org/apache/kafka/common/errors/OutdatedBrokerException.java

```diff
@@ -13,18 +13,18 @@
 package org.apache.kafka.common.errors;
 
 /**
- * Indicates that a request cannot be completed because an obsolete broker
+ * Indicates that a request cannot be completed because an outdated broker
  * does not support the required functionality.
```

Comment:
I was thinking about this and it seems that we can also get this error if the client requires a protocol version that is too old for the broker. If that's correct, then maybe the name could be misleading.